src/client/logger.hpp

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include <string>
+#include <memory>
+#include "inspector/content_log_domain.hpp"
+
+// Forward declarations
+class TrClientContextPerProcess;
+class ContentCdpHandler;
+
+/**
+ * Client-side logger that sends log messages to DevTools via CDP Log domain
+ * Singleton pattern for global access
+ */
+class Logger
+{
+public:
+  // Log levels matching CDP Log.LogEntry
+  enum class Level
+  {
+    VERBOSE,
+    INFO,
+    WARNING,
+    ERROR
+  };
+
+  // Log sources matching CDP Log.LogEntry
+  enum class Source
+  {
+    XML,
+    JAVASCRIPT,
+    NETWORK,
+    CONSOLE_API,
+    STORAGE,
+    APPCACHE,
+    RENDERING,
+    SECURITY,
+    DEPRECATION,
+    WORKER,
+    VIOLATION,
+    INTERVENTION,
+    RECOMMENDATION,
+    OTHER
+  };
+
+  // Singleton access
+  static Logger *getInstance();
```

Comment:
```suggestion
  static Logger *GetInstance();
```
static methods and fields must be upper case